docs/docs/userguide/configuration/BITBUCKET_CONFIGURATION.md

```diff
@@ -50,6 +49,13 @@ bitbucket_admin_email_address = "<EMAIL ADDRESS>"
 
     Please refer to [Sensitive Data](#sensitive-data) section.
 
+### Display Name
+Set the display name of the Bitbucket instance. Note that this value is only used during installation and changing the value during an upgrade has no effect.
```

Comment:
Out of interest what does setting the display name actually do - is it a UI feature? Maybe the comment should describe its purpose.

Reply:
yup, will elaborate it more. And display name will be showing up in the browser tab: 
<img width="560" alt="Screen Shot 2022-03-09 at 2 26 57 pm" src="https://user-images.githubusercontent.com/19516621/157367094-71d78fcc-247c-4b83-beeb-185355546176.png">

Reply:
This is the description in the documentation (but I don't think it is great). https://confluence.atlassian.com/bitbucketserver/automated-setup-for-bitbucket-776640098.html

```
The display name for the Bitbucket application.
```

I think providing more detail can be useful.

Reply:
I tried to find some official document on `display name`, but only found this one, very similar to what we got now: 
https://confluence.atlassian.com/bitbucketserver/automated-setup-for-bitbucket-776640098.html 

I'm thinking while we're not sure what other places will use `display name`, will it be misleading to try defining it here?